src/buildercore/core.py

```diff
@@ -221,6 +217,53 @@ def _all_nodes_filter(stackname, node_ids):
         query['instance-id'] = node_ids
     return query
 
+#
+#
+#
+
+def rds_dbname(stackname, context=None):
+    # TODO: investigate possibility of ambiguous RDS naming here
+    context = context or {}
+    return context.get('rds_dbname') or slugify(stackname, separator="") # *must* use 'or' here
+
+def rds_iid(stackname):
+    max_rds_iid = 63 # https://docs.aws.amazon.com/cli/latest/reference/rds/create-db-instance.html#options
+    # the rds iid needs to be deterministic, or, we need to find an attached rds db without knowing it's name
+    slug = slugify(stackname)
+    ensure(len(slug) <= max_rds_iid, "a database instance identifier must be less than 64 characters")
+    return slug
+
+def find_rds_instances(stackname, state='available'):
+    "This uses boto3 because it allows to start/stop instances"
+    try:
+        conn = boto_conn(stackname, 'rds', client=True) # RDS has no 'resource'
```

Comment:
I think the failure in https://alfred.elifesciences.org/job/process-stop-stacks-long-term/57/console is related. Will leave it there to avoid conflicts